woocommerce/woocommerce-ios
# [Mobile Payments] [SDK 2] Switch to SDK 2

This PR merges the long-lived feature/stripe-sdk-2 branch to develop. This will transition the app from SDK 1.x to SDK 2.x. This adds support for the M2 reader, changes the reader software updates to include mandatory un-cancellable updates, and other fixes.

To Test:

- Ensure you can
    - connect to a Chipper reader
    - connect to a M2 reader - see known issue #5326
    - choose from multiple discovered readers
    - connect to a reader after changing your store’s address in wp-admin (e.g. street address)
    - update a reader’s software - see known issues #5324 and #5325 
    - gracefully handle a reader with too low a battery to update - see known issue #5327
    - collect a tap payment
    - collect a chip (insert) payment
    - collect a swipe payment

Update release notes:
